string/string_test.mbt

```diff
@@ -40,10 +40,42 @@ test "compare" {
 }
 
 test "to_bytes" {
-  assert_eq!("中文".to_bytes().to_string(), "中文")
-  assert_eq!("asdf".to_bytes().to_string(), "asdf")
-  assert_eq!("🤣🤣".to_bytes().to_string(), "🤣🤣")
-  assert_eq!("🤔".to_bytes().to_string(), "🤔")
+  inspect!(
+    "中文".to_bytes(),
+    content=
+      #|b"-N\x87e"
+    ,
+  )
+  inspect!(
+    "\n\t\"\\\r\b".to_bytes(),
+    content=
+      #|b"\n\x00\t\x00\"\x00\\\x00\r\x00\b\x00"
```

Comment:
I'm still not quite sure whether `\n` should yield `\\n` or `\n` like this

Reply:
`\\n` should never be used unless the user absolutely wants to see the backslash, which is typically _not_ the case.